source/org/zfin/framework/presentation/DatabaseInfoController.java

```diff
@@ -21,10 +21,7 @@
 import javax.naming.NamingException;
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServletRequest;
-import java.io.BufferedReader;
 import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
```

Comment:
Should these have been removed?

Reply:
It should be safe to do so. They are not used in the file. I usually remove unused imports when IntelliJ points them out to me just to leave the file a little cleaner than when I found it.